python/delta_sharing/rest_client.py

```diff
@@ -330,7 +330,9 @@ def list_table_changes(self, table: Table, cdfOptions: CdfOptions) -> ListTableC
             metadata_json = json.loads(next(lines))
             actions: List[FileAction] = []
             for line in lines:
-                actions.append(FileAction.from_json(json.loads(line)))
+                file_action = FileAction.from_json(json.loads(line))
+                if (file_action is not None):
```

Comment:
Consider adding a comment on when can this be None .

Reply:
this change is reverted.